README.md

```diff
@@ -77,10 +98,13 @@ To avoid that, you can manually group errors specifying the error key that you w
 func main() {
 	c := periskop.NewErrorCollector()
 	req, err := http.NewRequest("GET", "http://example.com", nil)
-	c.ReportWithHTTPRequest(err, req, "example-request-error")
+	c.Report(periskop.ErrorReport{
+		err:         err,
+		httpRequest: req,
+		errKey:      "example-request-error",
+	})
 }
 ```
-__Note:__ With this method you are also aggregating by _error class_ which means that for the previous example the aggregation key is `*url.Error@example-request-error`.
```

Comment:
This is no longer true since [this commit](https://github.com/periskop-dev/periskop-go/commit/a844c9fc806858449424cdd49ddbfaa32362d647).